cockatrice/src/window_main.cpp

```diff
@@ -337,7 +337,7 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
             formattedMissingFeatures = "Missing Features: ";
```

Comment:
How do I add the missing tr() here?

`formattedMissingFeatures = "tr("Missing Features:) + QString:(" ")";` ?